app/views/products/_filter_test_link.html.erb

```diff
@@ -0,0 +1,16 @@
+<%
+
+# local variable 'task'
+
+%>
+
+<% if task.passing? %>
```

Comment:
Not a huge deal seeing how its a small amount of code but might consider revising to use a helper of some sort for this.  All that seems to be changing are a couple of styles and the link name